### ISSUE TITLE
[User experience] allow setting env variables from var tab

### ISSUE THREAD

**Issue by lared** [usebruno/bruno#379]
A lot of the time (almost always tbh) vars are used to transfer data which is specific to environment, like ID of user or auth token.

**Comment by helloanoop** [comment]
Did you checkout declarative scripting of response vars : https://docs.usebruno.com/scripting/vars.html

The post response vars are basically collection level var.

PS:
The ideas around request vars, collection vars, env vars are not very clear and also needs some additional dev to fully complete them. Will write  it in detail later today

**Comment by jeromelaurens** [comment]
I second this.
Why ? because I can switch very often between environments, then I should be able to set Vars on the current environment, not only on a collection level. Currently if I swich environment, the var is set with the value of the response from the previous environment. 
This is not what I wnat, especially for tokens, user Ids, and so on...

**Comment by David-Lannion** [comment]
This issue seems quite related with #292

**Comment by jeromelaurens** [comment]
> This issue seems quite related with #292

I don't think so. What does this issue has to do with nested environments ? Here we are talking about setting environment variables from var tab. Nothing to do with nested environment.
Am I wrong ? Did I misunderstood the #292  ?

**Comment by David-Lannion** [comment]
I put this comment because of the usage described in all comments.

**Comment by jeromelaurens** [comment]
hmm, I don't get it. 
the UX problem here, is that you have to go through scripting when you have to set an environment variable (and not collection variable). It would be be much simpler in the `Vars` tab, particularly in `Post Response` to be able to chose if the var is collection or environment wide. Still don't get how nested environment would simplify this.

**Comment by David-Lannion** [comment]
I did not intent to say "It is the same thing" but that both issues impact each other

**Comment by helloanoop** [comment]
Idea:

Currently, post-response variables update runtime variables.
We could enhance this by adding another column called Type, allowing users to specify the variable scope—Environment, Collection, or Runtime.

<img width="697" alt="image" src="https://github.com/user-attachments/assets/3a4c2429-c108-4cf3-9c95-422ff2e1af3c">

**Comment by anusree-bruno** [comment]
I think this would resolve this use case
https://github.com/usebruno/bruno/issues/3430

where you can right click select a value and choose a variable and scope to save this value to
